plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/messages.properties

```diff
@@ -0,0 +1,4 @@
+OPT_IN_DIALOG_TITLE=Google Eclipse Plugin
+OPT_IN_DIALOG_TEXT=Do you agree to send anonymous usage statistics of Google Eclipse Plugin to Google?
```

Comment:
This could be used by other plugins, like the Google Dataflow plugin too. Right?